scripts/prepare-aap.sh

```
@@ -41,4 +41,9 @@ oc set env deployment/osac-operator-controller-manager \
     -n ${INSTALLER_NAMESPACE} \
     OSAC_AAP_URL="${AAP_URL}/api/controller"
 
+# Set the correct AAP URL on the bare metal fulfillment operator deployment (triggers rollout)
+oc set env deployment/bmf-operator-controller-manager \
+    -n ${INSTALLER_NAMESPACE} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)